pset-functions.js

```diff
@@ -174,6 +228,16 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 4:')
 
 // Add your code below this line
+function isPalindrome(word) {
+  let reverseString = word.split("").reverse().join("")
+  if (word === reverseString) {
+    console.log("true")
+  } else {
+    console.log("false")
+    }
+}
+
+isPalindrome("racecar")
```

Comment:
@laraann88 Nice work here with this solution. I see where you chained together those methods on line 232 which was pretty good. One suggestion where you consle.log() true or false your can instead return false or true, then `console.log(isPalindrome("racecar))`

Check out this solution [here](https://github.com/jsr20190826/hw-02-functions-solution/blob/a4d789161886ff115657133c279062df8a4ca72d/pset-functions.js#L242)